lib/stripe/stripe_object.rb

```diff
@@ -43,7 +43,7 @@ def inspect
 
     def refresh_from(values, opts, partial=false)
       @opts = opts
-      @previous_metadata = values[:metadata]
+      @previous = values
```

Comment:
Oh, one other thought on naming -- I think `@original_values` would be a little clearer (`previous_values` makes me think it holds the values from a previous refresh or something)